src/app/modules/bounce/bounce.model.ts

```diff
@@ -58,26 +59,6 @@ const BounceSchema = new Schema<IBounceSchema>({
 })
 BounceSchema.index({ expireAt: 1 }, { expireAfterSeconds: 0 })
 
-// Helper function for methods.
-// Extracts custom headers which we send with all emails, such as form ID, submission ID
-// and email type (admin response, email confirmation OTP etc).
-const extractHeader = (body: IEmailNotification, header: string): string => {
-  return get(body, 'mail.headers').find(
-    (mailHeader) => mailHeader.name.toLowerCase() === header.toLowerCase(),
-  )?.value
-}
-
-// Helper function for methods.
-// Whether a bounce notification says a given email has bounced
-const hasEmailBounced = (
-  bounceInfo: IBounceNotification,
-  email: string,
-): boolean => {
-  return get(bounceInfo, 'bounce.bouncedRecipients').some(
-    (emailInfo) => emailInfo.emailAddress === email,
-  )
-}
-
```

Comment:
line 76 below:
`const bounces: ISingleBounce[] = get(snsInfo, 'mail.commonHeaders.to')`
Please add `[]` as a default for `get`?

Reply:
as above